odoo/models.py

```diff
@@ -3824,7 +3824,9 @@ def blacklist_given_fields(model):
         return [default]
 
     @api.multi
-    def copy_translations(old, new):
+    def copy_translations(old, new, excluded_fields=None):
+        excluded_fields = excluded_fields or []
```

Comment:
- I don't like long names, isn't `excluded` enough?
- Document what `excluded` is for.
- Use `()` as a default value.